### PR TITLE
New/Update Rules

### PR DESCRIPTION
**Pull request by nasbench**
- Renamed "sql_injection_keywords.yml" to "web_sql_injection_keywords.yml" to conform with the rest of the rule in the WEB directory
- Renamed "xss_keywords.yml" to "web_xss_keywords.yml" to conform with the rest of the rule in the WEB directory
- Renamed "proc_create_win_msdt_susp_parent.yml" to "proc_creation_win_msdt_susp_parent.yml" to conform with other process creation rules
-  Renamed "proc_create_win_sdiagnhost_susp_child.yml" to "proc_creation_win_sdiagnhost_susp_child.yml" to conform with other process creation rules
- Moved the rule "win_powershell_snapins_hafnium.yml" to process_creation folder instead of the WEB folder
- Created "web_susp_windows_path_uri.yml" to detect URI that contains susp windows paths
- Updated the description "web_webshell_keyword.yml" and added 3 more cases
-  Created "file_event_win_cve_2021_44077_poc_default_files.yml" to detect the default dropped file from the POC of CVE-2021-44077 (Showcased in the DFIR report)
- Created "proc_creation_win_renamed_plink.yml" to detect renamed usage of "Plink"

### CODE REVIEW

File: rules/web/web_susp_windows_path_uri.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    - https://thedfirreport.com/2022/06/06/will-the-real-msiexec-please-stand-up-exploit-leads-to-data-exfiltration/
+logsource:
+    category: webserver
+detection:
+    selection:
+        c-uri|contains:
+            - '=C:/Users'
+            - '=C:/Program%20Files'
+            - '=C:/Windows'
+    condition: selection
```
Comment (frack113) [inline rules/web/web_susp_windows_path_uri.yml:17]: in Catalina.txt log `%3A%5C` : 
can be 
```yaml
- '=C:/Users'
- '=C%3A%5CUsers'
```

File: rules/windows/process_creation/proc_creation_win_renamed_plink.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+title: Execution Of Renamed Plink Binary
+id: 1c12727d-02bf-45ff-a9f3-d49806a3cf43
+status: experimental
+description: Execution of a renamed version of the Plink binary
+author: Nasreddine Bencherchali
+references:
+    - https://thedfirreport.com/2022/06/06/will-the-real-msiexec-please-stand-up-exploit-leads-to-data-exfiltration/
```
Comment (frack113) [inline rules/windows/process_creation/proc_creation_win_renamed_plink.yml:7]: What is Plink  `- https://the.earth.li/~sgtatham/putty/0.58/htmldoc/Chapter7.html`

Comment (Neo23x0) [reply rules/windows/process_creation/proc_creation_win_renamed_plink.yml:7]: Fixed that one

File: rules/web/web_susp_windows_path_uri.yml (hunk 1 above)
Comment (Neo23x0) [reply rules/web/web_susp_windows_path_uri.yml:20]: I have extended the selection

**Pull request merged**